_posts/2021-01-25-solidity-developer-survey-2020-results.md

```diff
@@ -0,0 +1,359 @@
+- Compact code size
+- Syntax
+- Steady improvement of the language
+
+
+#### Most Dreaded or Disliked Features
+
+The currently most dreaded aspect about Solidity is clearly debugging, followed by a lack of Solidity libraries. 
+
+Respondents also complain about inheritance conflicts and the need to write more specific code in assembly / low-level code. 
+
+Other aspects and features that have been mentioned repeatedly as dreaded or disliked are:
+- Error handling
+- Memory management
+- Frequency of breaking changes and dependencies breaking
+- Contract & stack size limits
+- Stack too deep
+- Inconsistency with other languages
+- ABI encode
+- Unclear gas usage / gas inefficiency
+- Tooling
+- Complexity
+- Recompiles failing due to metadata
+- Verbosity
+- Dynamic array handling
+- Testing
+
+#### Most Anticipated Features
+
+The most anticipated feature by far was "no more SafeMath". SafeMath by default has been [introduced with 0.8.0](https://blog.soliditylang.org/2020/12/16/solidity-v0.8.0-release-announcement/) and hence is not an issue anymore. 🥳
+
+The second most anticipated feature is better debugging and more debugging tools. 
+
+Also ranked high was better documentation, especially with a focus on more tutorials and code examples.
+
+Other anticipated and frequently mentioned features were:
+- Gas efficiency optimization features
+- Generics
+- console.log()
+- Dynamic arrays
+- Better tooling
+- Better IDEs
+- LSP support
+- Structs
+- Rich revert reasons/ error codes
+- Standard library
+- Custom optimization features
+- Release v.1.0.0
+- SMTChecker
+- Stable ABIEncoderV2
+- No more stack too deep
+- Decimals
+- Over- and underflow handling
+- Fixed point types
+- WASM
+
+### (5) Community
+
+More than 55% of respondents interact with other Solidity developers on a regular basis, while 27% only claim to do so rarely. Almost 17% do not interact with other Solidity developers at all.
+
+![Developer Interaction](/img/2021/01/devinteraction.png)
+
+The majority of respondents feels welcome in the Solidity community and is confident in the work of the Soldity core team. 
+
+71% agree or somewhat agree to feeling welcome to contributing to Solidity.
+
+Only 22% fully agree that the Solidity team understands their needs. 14% state that they do not agree and don't feel understood by the Solidity team with regards to their needs.
+
+The process of how to contribute to Solidity is clear or somewhat clear to 41% of respondents while 32% do not agree and hence say that the contribution process is unclear to them. 
+
+⚠️ _Note that there was a comparatively high share of "I don't know" replies to the last three statements._
+
+![Community Statements](/img/2021/01/communitystatements.png)
+
+#### Language Design
+
+Almost 85% of respondents state that they have never participated in any Solidity language design related activity. 2/3 of them say they didn't do so because they don't know how, while 1/3 is simply not interested in it.
```

Comment:
```suggestion
Almost 85% of respondents state that they have never participated in any Solidity language design related activity. Around two third of them say they didn't do so because they don't know how, while one third is simply not interested in it.
```